test/lib/validation/schema/__helper__/extension.js

```diff
@@ -0,0 +1,73 @@
+const customConfiguration = require("./customConfiguration");
+
+/**
+ * Common test functionality to be able to run the same tests for different types of kind "extension"
+ */
+module.exports = {
+	/**
+	 * Executes the tests for different types of kind extension, e.g. "project-shim", "server-middleware" and "task"
+	 *
+	 * @param {Function} test ava test
+	 * @param {Function} assertValidation assertion function
+	 * @param {string} type one of "project-shim", "server-middleware" and "task"
+	 * @param {Object} additionalConfiguration additional configuration content
+	 */
+	defineTests: function(test, assertValidation, type, additionalConfiguration) {
+		additionalConfiguration = additionalConfiguration || {};
+		additionalConfiguration = Object.assign({"kind": "extension"}, additionalConfiguration);
+
+		customConfiguration.defineTests(test, assertValidation, type, additionalConfiguration);
+
+		["2.0", "2.1"].forEach((specVersion) => {
+			test(`kind: extension / type: ${type} basic (${specVersion})`, async (t) => {
+				await assertValidation(t, Object.assign({
+					"specVersion": specVersion,
+					"kind": "extension",
```

Comment:
Duplicate information. Already defined in `additionalConfiguration`, so should be removed here
```suggestion
```

Reply:
Done